pandas/tests/plotting/test_groupby.py

```diff
@@ -10,11 +10,52 @@
 import numpy as np
 
 from pandas.tests.plotting.common import TestPlotBase
+import pytest
 
 
 @td.skip_if_no_mpl
 class TestDataFrameGroupByPlots(TestPlotBase):
 
+    @pytest.mark.parametrize('equal_bins, bins, expected1, expected2', (
```

Comment:
Need two `expected` variables for the two groups. They correspond to the leftmost part of each histogram bar on the x-axis.